pkgs/by-name/sp/speakersafetyd/package.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchCrate
+, pkg-config
+, alsa-lib
+, rust
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "speakersafetyd";
+  # tracking: https://src.fedoraproject.org/rpms/rust-speakersafetyd
+  version = "0.1.9";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-I1fL1U4vqKxPS1t6vujMTdi/JAAOCcPkvUqv6FqkId4=";
+  };
+  cargoHash = "sha256-Adwct+qFhUsOIao8XqNK2zcn13DBlQNA+X4aRFeIAXM=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ alsa-lib ];
+
+  postPatch = ''
+    substituteInPlace speakersafetyd.service --replace "/usr" "$out"
+    substituteInPlace Makefile --replace "target/release" "target/${rust.lib.toRustTargetSpec stdenv.hostPlatform}/$cargoBuildType"
+  '';
+
+  installFlags = [
+    "BINDIR=$(out)/bin"
+    "UNITDIR=$(out)/lib/systemd/system"
+    "UDEVDIR=$(out)/lib/udev/rules.d"
+    "SHAREDIR=$(out)/share"
+    "TMPFILESDIR=$(out)/lib/tmpfiles.d"
+    "VARDIR=$(PWD)" # creating files in /var does not make sense in a nix package
```

Comment:
Could we substitute/sed that line out instead?  Otherwise, if they start putting files there for some reason rather than just an empty directory, we'll silently lose them.

Reply:
Done